### DISCUSSION TITLE
What's the difference between the advanced and regular PCB?

### DISCUSSION THREAD

**Discussion by KuestenKeks** [HDR/Gameboy-Camera-Flashcart#3] [Q&A] [answered]
The description on of the advanced PCB on PCBWay states "This is the Advanced PCB Order version only order this version if you know what you're doing!". Otherwise the descriptions are identical.

I clearly don't know what I'm doing, but I'd like to know the difference anyway :D

Cheers :)

**Comment by HDR** [comment, marked as answer]
Hi, the Advanced PCB option is for people that want the more durable hard gold finish on their carts, this is what is used on original cartridges, but it's very expensive (200-400 usd depending on the selected options)

**Comment by KuestenKeks** [comment]
Some further explanations of the differences: https://www.sharrettsplating.com/blog/hard-gold-enig/
